server/index.js

```diff
@@ -19,6 +19,23 @@ app.get('/api/health-check', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/products', (req, res, next) => {
+  const sql = `select "productId",
+                      "name",
+                      "price",
+                      "image",
+                      "shortDescription"
+               from "products"
+                `;
+
+  db.query(sql)
+    .then(result => {
+      const products = result.rows;
+      res.status(200).send(products);
+    })
+    .catch(err => res.status(500).send('Query error!', err));
```

Comment:
We never send our errors to the client, this is a security concern!

Instead, just `console.error()` errors and send a generic message